CMSgov/onemac
# Helper lambda: getPackageItems

Story: none
Endpoint: See github-actions bot comment

### Details
We've really manipulated the data model and need to sort out the seed data to get the automated tests working again... getPackageItems is a small utility function to output as json all items for a package.  Should make building the seed data easier...

### Changes
- added the getPackageItems lambda

### Test Plan
1. Log into the AWS console
2. Find a package in the database
3. Navigate to Lambdas/Functions  and select the getPackageItems
4. run the lambda with { "packageId": "<the id>" } as the event
5. Verify that all items for that package are included in the output json.
